tests/test_reify.cpp

```diff
@@ -0,0 +1,122 @@
+#include <catch2/catch_test_macros.hpp>
+#include <potassco/aspif_text.h>
+#include <potassco/reify.h>
+
+#include <sstream>
+
+namespace Potassco::Test::Reify {
+bool read(std::stringstream& in, std::stringstream& out, bool scc = false, bool step = false) {
```

Comment:
This function should not have external linkage. Please consider making it `static`.